src/Essentials/src/Flashlight/Flashlight.android.cs

```diff
@@ -58,7 +58,7 @@ Task ToggleTorchAsync(bool switchOn)
 			{
 				lock (locker)
 				{
-					if (OperatingSystem.IsAndroidVersionAtLeast((int)BuildVersionCodes.M) && !AlwaysUseCameraApi)
+					if (OperatingSystem.IsAndroidVersionAtLeast(23) && !AlwaysUseCameraApi)
```

Comment:
The analyzer could not track a variable value, could we use a version numbers directly?

Reply:
Yes. Better to be consistent.